src/main/java/org/springframework/data/couchbase/repository/CouchbaseRepository.java

```diff
@@ -43,4 +47,8 @@
 	@Override
 	List<T> findAllById(Iterable<ID> iterable);
 
+	CouchbaseEntityInformation<T, String> getEntityInformation();
```

Comment:
is it ok to add this here @mp911de ?

Reply:
Yes. I will move it to SimpleCouchbaseRepository and SimpleReactiveCouchbaseRepository.
(oddly, there is not option for me to add a comment here, but I can edit yours).

Edit: I will leave this here - I tried moving it to SimpleCouchbaseRepository, but casting the 'target' in DynamicInvocationHandler to a SimpleCouchbaseRepository results in a ClassCastException due to being loaded in a different class loader.